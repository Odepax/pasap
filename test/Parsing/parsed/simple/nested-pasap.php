<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Document</title>
	</head>
	<body>
		<article>
			<h1>Some Pasap Custom Tags</h1>
			<four>
				<one/>
				<five/>
				<six>
					<two/>
					<three/>
				</six>
			</four>
		</article>
	</body>
</html>
