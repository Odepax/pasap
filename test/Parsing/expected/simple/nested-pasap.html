<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Document</title>
	</head>
	<body>
		<article>
			<h1>Some Pasap Custom Tags</h1>
			<div class="Four">
				<h5>Four</h5>
				<div class="children">
					<h5>One</h5>
					<div class="Five">
						<h5>Five</h5>
						<div class="children"></div>
					</div>
					<div class="Six">
						<h5>Six</h5>
						<div class="children">
							<h5>Two</h5>
							<h5>Three</h5>
						</div>
					</div>
				</div>
			</div>
		</article>
	</body>
</html>
